src/core/exec-invoke.c

```diff
@@ -5265,12 +5265,12 @@ int exec_invoke(
         log_command_line(context, params, "Executing", executable, final_argv);
 
         if (params->exec_fd >= 0) {
-                uint8_t hot = 1;
+                usec_t t = now(CLOCK_MONOTONIC);
 
                 /* We have finished with all our initializations. Let's now let the manager know that. From this point
                  * on, if the manager sees POLLHUP on the exec_fd, then execve() was successful. */
 
-                if (write(params->exec_fd, &hot, sizeof(hot)) < 0) {
+                if (write(params->exec_fd, &t, sizeof(t)) < 0) {
```

Comment:
I can't say I am enthusiastic about the idea that exec_fd is now overloaded for a completely different purpose. Please add a comment to the exec_fd declaration that it has two purposes.